### ISSUE TITLE
Asus ux333FA 

### ISSUE THREAD

**Issue by ressanoid** [hieplpvip/ASUS-ZENBOOK-UX410-HACKINTOSH#50]
**What is your laptop?**
Asus UX333FA-AB77
**Full specs**
Intel Core i7-8565u Quad Core Processor (Up to 4.6GHz) 
 Intel HD 
16GB RAM
13.3" FHD  screen
512GB PCIe SSD
TPM Security Chip
1x HDMI, 
1x USB Type C,
2xUSB ports
gigabit-class Wi-Fi 5 (802.11AC),
Bluetooth 5.0 
1x Micro SD card reader

**Attach original DSDT from Clover**
[origin.zip](https://github.com/hieplpvip/ASUS-ZENBOOK-HACKINTOSH/files/3613643/origin.zip)

**Attach full report from AIDA64 (if possible)**
I don't know how to do it

**Comment by ressanoid** [comment]
Intel(R) UHD Graphics 620

Report Date: Wednesday, September 18, 2019
Report Time [hh:mm:ss]: 8:06:09 AM
Driver Version: 25.20.100.6577
Operating System: Windows* 10 Pro (10.0.18362)
Physical Memory: 16198 MB
Vendor ID: 8086
Device ID: 3EA0
Device Revision: 00
Graphics Output Protocol (GOP) Version: 9.0.1085
Current Resolution: 1920 x 1080

Processor: Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
Processor Speed: 1992 MHz
Processor Graphics in Use: Intel(R) UHD Graphics 620
Shader Version: 5.1
OpenGL* Version: 4.5
OpenCL* Version: 2.1
Vulkan* Version: 1.1.97


* Microsoft DirectX* *
Runtime Version: 12.0
Hardware-Supported Version: 12.0


* Devices connected to the Graphics Accelerator *

Active Displays: 1

* Built-in Display *
Display Type: Digital
DDC2 Protocol: Supported
Gamma: 2.2
Connector Type: Embedded DisplayPort
Device Type: Built-in Display


Maximum Image Size
Horizontal Size: 11.42 inches
Vertical Size: 6.69 inches

Supported Modes
1920 x 1080 (60p Hz)
1920 x 1080 (48p Hz)

Raw EDID:
00 FF FF FF FF FF FF 00 06 AF 2D 5C 00 00 00 00 
14 1C 01 04 A5 1D 11 78 02 EE 95 A3 54 4C 99 26 
0F 50 54 00 00 00 01 01 01 01 01 01 01 01 01 01 
01 01 01 01 01 01 B4 37 80 A0 70 38 3E 40 3A 2A 
35 00 25 A5 10 00 00 18 B4 37 80 A0 70 38 5C 41 
3A 2A 35 00 25 A5 10 00 00 18 00 00 00 FE 00 41 
55 4F 0A 20 20 20 20 20 20 20 20 20 00 00 00 FE 
00 42 31 33 33 48 41 4E 30 35 2E 43 20 0A 00 AC

**Comment by gabriel-viviani** [comment]
@ressanoid did you manage to run osx in this laptop?

**Comment by ressanoid** [comment]
> 
> 
> @ressanoid did you manage to run osx in this laptop?

yes it's working but kind of laggy a bit I'll try open core on it sometime later today and see if it works
